tools/scripts/ValidateReleaseInfo.ps1

```diff
@@ -75,7 +75,7 @@ function ValidateChannelInfo($channel)
 }
 
 # Load the octokit dll
-Add-Type -Path ((Get-Location).Path + '\Octokit.0.32.0\lib\net45\Octokit.dll')
+Add-Type -Path ((Get-Location).Path + '\Octokit.0.33.0\lib\net45\Octokit.dll')
```

Comment:
This wouldn't normally be needed, but OctoKit was updated. I'll create a feature to make us more resilient to version changes